utils/certs/gen_certificates.sh

```diff
@@ -115,10 +115,10 @@ extendedKeyUsage = clientAuth
 function generate_ca_cert () {
     echo "Generating Private CA Root Certificate"
     # Generate Private key and set permissions
-    openssl genrsa -out "${PRIVATE}/daosCA.key" 4096
+    openssl genrsa -out "${PRIVATE}/daosCA.key" 3072
     chmod 0400 "${PRIVATE}/daosCA.key"
     # Generate CA Certificate
-    openssl req -new -x509 -config "${CA_HOME}/ca.cnf" -days 1095  -sha512 \
+    openssl req -new -x509 -config "${CA_HOME}/ca.cnf" -days 365  -sha512 \
```

Comment:
Why is the CA cert only valid for a year?

Reply:
I believe this is considered best practice for internally run and managed CAs.